shoop/admin/generated_resources.txt

```diff
@@ -0,0 +1,9 @@
+static/shoop_admin/js/picotable.js
```

Comment:
Anyway, header would be nice still. The parser already ignores lines starting with a `#`, so I'll add a header.